aws-android-sdk-appsync-tests/src/androidTest/java/com/amazonaws/mobileconnectors/appsync/AppSyncTestSetupHelper.java

```diff
@@ -94,6 +94,16 @@ public String getS3Region() {
         return s3Region;
     }
 
+    public AWSAppSyncClient createAppSyncClientWithApiKey() {
```

Comment:
Either accept SubscriptionIntegrationTestGogi through a parameter or rename function to include SubscriptionIntegrationTestGogi